csrc/kernel_cache.h

```diff
@@ -283,7 +283,7 @@ class TORCH_CUDA_CU_API InputsIdLookup : public NonCopyable {
   //! depend on those inputs and omitting them would lead to a collision.
   IdLookupReturn lookupId(
       const at::ArrayRef<c10::IValue>& inputs,
-      bool hash_scalars = false);
+      const std::vector<bool>* input_affects_concretization = nullptr);
```

Comment:
Please update the function comment above.

Reply:
Why is this a pointer to a vector? Why nullptr is allowed?

Reply:
Added comment to explain this argument. `nullptr` is allowed since lookupId is also used for a higher-level cache in the frontend which cannot know what inputs affect concretization, so must include them all: https://github.com/NVIDIA/Fuser/blob/main/csrc/python_frontend/fusion_cache.cpp#L203.